renderdoc/driver/vulkan/vk_info.cpp

```diff
@@ -690,10 +690,12 @@ void VulkanCreationInfo::RenderPass::Init(VulkanResourceManager *resourceMan,
 
     dst.inputAttachments.resize(src.inputAttachmentCount);
     dst.inputLayouts.resize(src.inputAttachmentCount);
+    dst.inputStencilLayouts.resize(src.inputAttachmentCount);
     for(uint32_t i = 0; i < src.inputAttachmentCount; i++)
     {
       dst.inputAttachments[i] = src.pInputAttachments[i].attachment;
       dst.inputLayouts[i] = src.pInputAttachments[i].layout;
+      dst.inputStencilLayouts[i] = VK_IMAGE_LAYOUT_UNDEFINED;
```

Comment:
Should this not default to the same value as `dst.inputLayouts[i]`, then get overwritten optionally with a separate layout? Analogous to how `stencilLayout` is initially the same as `depthLayout`, but then gets optionally overwritten.

Since enabling the separate depth/stencil feature is enough to have us track depth and stencil separately, but then there's nothing to stop an application from having a renderpass without separate depth/stencil layouts. We'd want to be sure we didn't transition to/from UNDEFINED.